README.md

```diff
@@ -1,16 +1,8 @@
-## Titanium CLI
+# Titanium CLI
 
-> [Titanium CLI](https://github.com/appcelerator/titanium) is a
-[Command Line Tool](http://en.wikipedia.org/wiki/Command-line_interface)
-for creating and building Titanium Mobile applications and modules. It's
-open-source and easy to use. [We've](https://github.com/appcelerator)
-designed Titanium to be suitable for command line beginners, but still be
-powerful and extensible enough for production usage.
+> [Titanium CLI](https://github.com/tidev/titanium) is a [Command Line Tool](http://en.wikipedia.org/wiki/Command-line_interface) for creating and building Titanium Mobile applications and modules. It's open-source and easy to use. [We've](https://github.com/tidev) designed Titanium to be suitable for command line beginners, but still be powerful and extensible enough for production usage.
 
-[![Build Status](https://travis-ci.org/appcelerator/titanium.svg?branch=master)](https://travis-ci.org/appcelerator/titanium)
-[![Dependency Status](https://david-dm.org/appcelerator/titanium.svg)](https://david-dm.org/appcelerator/titanium)
-[![devDependency Status](https://david-dm.org/appcelerator/titanium/dev-status.svg)](https://david-dm.org/appcelerator/titanium#info=devDependencies)
-[![NPM version](https://badge.fury.io/js/titanium.svg)](http://badge.fury.io/js/grunt-appc-js) [![Greenkeeper badge](https://badges.greenkeeper.io/appcelerator/titanium.svg)](https://greenkeeper.io/)
+[![NPM version](https://badge.fury.io/js/titanium.svg)](http://badge.fury.io/js/grunt-appc-js) [![Greenkeeper badge](https://badges.greenkeeper.io/tidev/titanium.svg)](https://greenkeeper.io/)
```

Comment:
Greenkeeper is dead, this can be removed.

Reply:
deleted